### PR TITLE
Add solidity linter

### PR DESCRIPTION
**Pull request by nrchandan**
After this change, build will fail if a commit doesn't follow the
linting rules. The linting rules can be changed in .soliumrc.json

### CODE REVIEW

**Comment by nrchandan** [comment]
@inovizz Please review

**Comment by inovizz** [comment]
:+1:

File: contracts/Parent.sol [hunk 1]
```diff
@@ -6,6 +6,7 @@ import "./OrganisationInterface.sol";
 
 import "./helper_contracts/zeppelin/ownership/Ownable.sol";
 
+
```
Comment (inovizz) [inline contracts/Parent.sol]: Is it a recommendation from solidity linter to have two blank lines before starting Contract Definition?

Comment (nrchandan) [inline contracts/Parent.sol]: Not a recommendation, but the expectation from this particular tool.

Comment (inovizz) [inline contracts/Parent.sol]: Ok.

**Pull request merged**